src/server/SocketServer.java

```diff
@@ -1,5 +1,11 @@
 package server;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.net.Socket;
+import java.net.ServerSocket;
+import java.io.IOException;
```

Comment:
Organize the imports. They should look like this

```suggestion
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import java.io.IOException;
import java.net.ServerSocket;
import java.net.Socket;
```